src/app/static/src/app/store/translations/locales.ts

```diff
@@ -173,6 +175,53 @@ export interface Translations {
 }
 
 const en: Translations = {
+    accessibility: "Accessibility",
+    accessibilityContent: "<h1>Accessibility on NAOMI</h1>\n" +
+        "                <p>This statement applies to content published on <a href=\"https://naomi.unaids.org/\">naomi.unaids.org</a></p>\n" +
+        "                <p>We want as many people as possible to be able to use this website. For example, that means you should\n" +
+        "                    be able to:</p>\n" +
+        "                <ul>\n" +
+        "                    <li>Resize your window with content being reformatted appropriately</li>\n" +
+        "                    <li>Adjust your text size without the site becoming less usable</li>\n" +
+        "                    <li>Read text easily due to sufficient contrast between foreground and background elements</li>\n" +
+        "                </ul>\n" +
+        "                <p><a href=\"https://mcmw.abilitynet.org.uk/\">AbilityNet</a> has advice on making your device easier to\n" +
+        "                    use if you have a disability.</p>\n" +
+        "                <h2>How accessible the website is</h2>\n" +
+        "                <p>Parts of this website may not be fully accessible. For example:</p>\n" +
+        "                <ul>\n" +
+        "                    <li>It may not be entirely navigable by keyboard</li>\n" +
```

Comment:
An issue that I suspect might come up is the lack of semantic code, which can make the code difficult to navigate for people using e-readers. The application isn't a typical web page so it can be tricky to apply regular patterns but I think we might be guilty of overusing the div element when other more semantic elements would have been appropriate.

Reply:
On point, was spotted during accessibility testing. A statement has been added.